app/views/state_file/questions/return_status/_rejected.html.erb

```diff
@@ -39,20 +39,26 @@
 <% end %>
 
 <% if @error&.auto_wait %>
-  <section class="spacing-below-15">
-    <p class="spacing-below-5"><strong><%= t('.next_steps.can_edit.title') %></strong></p>
-    <%= @error.resolution(I18n.locale).present? ? @error.resolution(I18n.locale) : t('.next_steps.can_edit.body') %>
-  </section>
+  <% if app_time.before?(Rails.configuration.state_file_end_of_in_progress_intakes) %>
+    <section class="spacing-below-15">
+      <p class="spacing-below-5"><strong><%= t('.next_steps.can_edit.title') %></strong></p>
+      <%= @error.resolution(I18n.locale).present? ? @error.resolution(I18n.locale) : t('.next_steps.can_edit.body') %>
```

Comment:
@tofarr from what tiffany said in slack I think we actually need to show the default text if its after this time so this `<%= @error.resolution(I18n.locale).present? ? @error.resolution(I18n.locale) : t('.next_steps.can_edit.body') %>` should actually be something more like this `<%= @error.resolution(I18n.locale).present? && app_time.before?(Rails.configuration.state_file_end_of_in_progress_intakes) ? @error.resolution(I18n.locale) : t('.next_steps.can_edit.body') %>`